ivadomed/models.py

```diff
@@ -1205,21 +1233,25 @@ def __init__(self, in_channels, gating_channels, inter_channels=None, dimension=
 
         # Theta^T * x_ij + Phi^T * gating_signal + bias
         self.theta = conv_nd(in_channels=self.in_channels, out_channels=self.inter_channels,
-                             kernel_size=self.sub_sample_kernel_size, stride=self.sub_sample_factor, padding=0,
-                             bias=False)
+                             kernel_size=self.sub_sample_kernel_size, stride=self.sub_sample_factor, padding=0, bias=False)
         self.phi = conv_nd(in_channels=self.gating_channels, out_channels=self.inter_channels,
                            kernel_size=1, stride=1, padding=0, bias=True)
-        self.psi = conv_nd(in_channels=self.inter_channels, out_channels=1, kernel_size=1, stride=1, padding=0,
-                           bias=True)
+        self.psi = conv_nd(in_channels=self.inter_channels, out_channels=1, kernel_size=1, stride=1, padding=0, bias=True)
 
         # Initialise weights
         for m in self.children():
-            m.apply(weights_init_kaiming)
-
+            weights_init_kaiming(m)
+         
         # Define the operation
         self.operation_function = self._concatenation
 
     def forward(self, x, g):
+        '''
+        :param x: (b, c, t, h, w)
+        :param g: (b, g_d)
+        :return:
```

Comment:
Thanks for adding this docstring. Would be great if you could further detail, eg c = channel + match the docstring style used in `ivadomed` repository.